metrics/metrics.go

```diff
@@ -37,12 +39,20 @@ var (
 	})
 )
 
-func recordMetrics() {
+func recordMetrics(errCh chan error) {
+	dbPath := config.Cfg.Common.ProjectDir + "/" + config.Cfg.UserID
+	db, err := metastore.NewDb(dbPath)
+	if err != nil {
+		return
+	}
 	go func() {
```

Comment:
Based on the above comment https://github.com/spinup-host/spinup/pull/98/files#r832724971, does that mean we are spawning a new go-routine every time when we hit `/metrics`?